hypothesis-python/tests/cover/test_control.py

```diff
@@ -27,15 +27,15 @@
     UnsatisfiedAssumption,
 )
 from hypothesis.internal.compat import ExceptionGroup
-from hypothesis.internal.conjecture.data import ConjectureData as TD
+from hypothesis.internal.conjecture.data import ConjectureData
 from hypothesis.stateful import RuleBasedStateMachine, rule
 from hypothesis.strategies import integers
 
 from tests.common.utils import capture_out
 
 
 def bc():
-    return BuildContext(TD.for_buffer(b""))
+    return BuildContext(ConjectureData.for_buffer(b""))
```

Comment:
noticed in passing; I hope you don't mind. I couldn't find usages of `TD` (TestData, presumably?) elsewhere.